src/foam/u2/view/UnstyledTableView.js

```diff
@@ -199,6 +203,7 @@ foam.CLASS({
       var columnSelectionE;
 
       if ( this.filteredTableColumns$ ) {
+        // TODO
```

Comment:
TODO what?

Reply:
Oh shoot, that was to remind myself to make sure this didn't break the feature where you can adjust the table columns. I'll check that.

Reply:
Fixed here: https://github.com/foam-framework/foam2/pull/3110